setup.py

```diff
@@ -33,43 +35,44 @@
         "Operating System :: OS Independent",
         "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
     ],
-    keywords='Python Plone',
-    author='Tiberiu Ichim',
-    author_email='tiberiu.ichim@eaudeweb.ro',
-    url='https://github.com/collective/eea.restapi',
+    keywords="Python Plone",
+    author="Tiberiu Ichim",
+    author_email="tiberiu.ichim@eaudeweb.ro",
+    url="https://github.com/collective/eea.restapi",
     project_urls={
-        'PyPI': 'https://pypi.python.org/pypi/eea.restapi',
-        'Source': 'https://github.com/collective/eea.restapi',
-        'Tracker': 'https://github.com/collective/eea.restapi/issues',
+        "PyPI": "https://pypi.python.org/pypi/eea.restapi",
+        "Source": "https://github.com/collective/eea.restapi",
+        "Tracker": "https://github.com/collective/eea.restapi/issues",
         # 'Documentation': 'https://eea.restapi.readthedocs.io/en/latest/',
     },
-    license='GPL version 2',
-    packages=find_packages('src', exclude=['ez_setup']),
-    namespace_packages=['eea'],
-    package_dir={'': 'src'},
+    license="GPL version 2",
+    packages=find_packages("src", exclude=["ez_setup"]),
+    namespace_packages=["eea"],
+    package_dir={"": "src"},
     include_package_data=True,
     zip_safe=False,
     python_requires="==2.7",
     install_requires=[
-        'setuptools',
+        "setuptools",
         # -*- Extra requirements: -*-
-        'z3c.jbot',
-        'collective.folderishtypes',
-        'plone.api>=1.8.4',
-        'plone.restapi',
-        'plone.app.dexterity',
-        'requests',
-        'xlsxwriter',
+        "z3c.jbot",
+        "collective.folderishtypes",
+        "plone.api>=1.8.4",
+        "plone.restapi",
+        "plone.app.dexterity",
+        "requests",
+        "xlsxwriter",
+        "eea.volto.slate",
```

Comment:
@tiberiuichim 

I wonder why is eea.restapi requiring `eea.volto.slate`? 

Anyway, you need to release it on eggrepo or pypi first.